collections/_article/godot-4-2-arrives-in-style.md

```diff
@@ -0,0 +1,426 @@
+- Renaming an audio bus no longer emits the `bus_layout_changed` signal in `AudioServer`; you can use the new `bus_renamed` signal instead ([GH-81641](https://github.com/godotengine/godot/pull/81641)).
+- The default base part of the Android package name was changed from `org.godotengine` to `com.example` ([GH-80761](https://github.com/godotengine/godot/pull/80761)).
+- A previously unutilized notification, `NOTIFICATION_NODE_RECACHE_REQUESTED`, was completely removed from the engine ([GH-84419](https://github.com/godotengine/godot/pull/84419)).
+
+
+### Core and systems
+
+Following the big changes to resource loading and multithreading done in Godot 4.1, the core team is more focused on stability this time around.
+
+This includes addressing design limitations when changing scenes or renaming nodes during `ready`, which led to crashes ([GH-78988](https://github.com/godotengine/godot/pull/78988), [GH-78706](https://github.com/godotengine/godot/pull/78706), [GH-85280](https://github.com/godotengine/godot/pull/85280)), as well as various problems related to renaming and moving files around ([GH-80503](https://github.com/godotengine/godot/pull/80503), [GH-81657](https://github.com/godotengine/godot/pull/81657)). The latter could also lead to corruption of scene files, which should no longer occur. Also, some cases of sporadic changing of resource IDs in scenes have been solved ([GH-65011](https://github.com/godotengine/godot/pull/65011)). There is still room for improvement, but this already makes 4.2 way more version control friendly.
+
+These issues were fixed thanks to submissions from veteran contributors, such as [Pedro J. Estébanez](https://github.com/RandomShaper), [RedworkDE](https://github.com/RedworkDE), and [Rindbee](https://github.com/Rindbee), as well as project newcomers who only started contributing recently, [OXTyler](https://github.com/OXTyler) and [Jordyfel](https://github.com/Jordyfel).
+
+A rather infamous issue, manifesting in crashes and exhaustive printing of `slot >= slot_max` to the output log, met its demise in Godot 4.2 at the hand of [Pāvels Nadtočajevs](https://github.com/bruvzg) ([GH-85280](https://github.com/godotengine/godot/pull/85280)). It was a pretty nasty bug which would only show up in export builds of your project and was related, one way or another, to manipulating the scene tree. And now it's thankfully gone.
+
+Sometimes a small mistake can result in major and unexpected differences in behavior. Such is the case with this bug that was spotted and fixed by the project's bugsquad powerhouse [AThousandShips](https://github.com/AThousandShips) ([GH-81037](https://github.com/godotengine/godot/pull/81037)). This issue caused abnormal memory usage and performance issues in seemingly trivial use cases. If you are using a lot of plain Godot objects, you should see a noticeable decrease in memory usage in 4.2.
```

Comment:
Could we mention the future proofing one too as it's directly related to this? 🙂 being: https://github.com/godotengine/godot/pull/81020

Reply:
Sure!